core/tests/test_utils.py

```diff
@@ -71,11 +72,28 @@ def __init__(self, service_name='urlfetch'):
         self.request = None
 
     def set_return_values(self, content='', status_code=200, headers=None):
+        """Sets the content, status code and headers that the urlfetch mock
+        should return with.
+
+        Args:
+            content: str. The content to return in subsequent calls to the
+            urlfetch mock.
+            status_code: int. The status_code to return in subsequent calls
+            to the urlfetch mock.
+            headers: dict. The headers to return in subsequent calls to
```

Comment:
I think headers would have some standard expected stuff as per HTTP protocol. Isn't it? -- What do you want to be added here exactly?